packages/mobile/src/identity/verification.ts

```diff
@@ -432,12 +425,6 @@ export function* requestAndRetrieveAttestations(
 
     // Check if we have a sufficient set now by fetching the new total set
     attestations = yield call(getActionableAttestations, attestationsWrapper, phoneHash, account)
-    if (shortVerificationCodesEnabled) {
-      // we only support attestation service 1.1.0 and above for short codes
-      attestations = attestations.filter(
-        (att) => !isVersionBelowMinimum(att.version, MINIMUM_VERSION_FOR_SHORT_CODES)
-      )
-    }
```

Comment:
Should we log if we receive attestations below the required version?

Reply:
I think there are no attestation services with that old a version. I asked in cLabs identity channel before deleting this https://valora-app.slack.com/archives/CJXJ1EV0D/p1637346367233800 and they said this code path is no longer supported